### PR TITLE
Group 3: West Coast BEST COAST - All tests passing!! Yay!! :) 

### PR DESCRIPTION
**Pull request by JessicaNgo**
anthony(pst please)
Stephen Spears
Jessica Ngo

Will clean up code/try to improve run time later

### CODE REVIEW

File: tests/resources_tests/test_followers.py [hunk 1]
```diff
@@ -9,6 +9,7 @@ def test_get_followers(self):
         # Preconditions
         headers = {'Authorization': '$RMOTR$-U2'}
         response = self.client.get('/followers', headers=headers)
+        # print (response.charset)
```
Comment (santiagobasulto) [inline tests/resources_tests/test_followers.py:12]: 😱

File: twitter_timeline/main.py [hunk 2]
```diff
@@ -23,22 +29,97 @@ def before_request():
 
 
 @app.route('/friendship', methods=['POST', 'DELETE'])
-@json_only
+#@json_only
```
Comment (santiagobasulto) [inline twitter_timeline/main.py]: If you take this one out, and someone sends a not-json request, your code is gonna break

File: twitter_timeline/main.py [hunk 3]
```diff
@@ -23,22 +29,97 @@ def before_request():
...
-
+    #Checking if there's a username in our request
+    try:
+        #
+        friend_username = request.json["username"]
+        friend_object = g.db.users.find_one({"username": friend_username})
+        friend_id = friend_object["_id"]
+        user_object = g.db.users.find_one({"_id": user_id})
+    except:
+        print("ERROR: NO USERNAME IN PAYLOAD. NO USERNAME IN DATABASE")
```
Comment (santiagobasulto) [inline twitter_timeline/main.py]: print?

Comment (JessicaNgo) [inline twitter_timeline/main.py]: oops! debugging with print messages :x I will go clean this code up!!

File: tests/resources_tests/test_followers.py (hunk 1 above)
Comment (JessicaNgo) [inline tests/resources_tests/test_followers.py:12]: sorry!! :ghost:

**Pull request closed without merging**